gemma-core/src/main/java/ubic/gemma/core/apps/AffyDataFromCelCli.java

```diff
@@ -44,9 +44,9 @@
 
     private static final String APT_FILE_OPT = "aptFile";
 
-    public static void main( String[] args ) {
+    public static int main( String[] args ) {
```

Comment:
We can get rid of all those main() methods altogether because GemmaCLI uses reflection to run `executeCommand`.